### ISSUE TITLE
Prepare Content: Convert text

### ISSUE THREAD

**Issue by jfrank-nih** [NCIOCPL/akumina-mynci#18]
**Top Line:** Create a new `convertText` method on the `DrupalContent` class.

_Input:_
* `text`: String text to convert
* `urlConverter`: Instance of the URL converter class

_Tasks:_
* Find any `<img>` or `<a>` tags in the text blob
   * Take the URL from the tag and run it through the URL converter #19
   * Replace the old URL with the returned new URL in the text
* May add additional conversions later

_Returns:_
* Converted text string

**Comment by erikscottmeyer** [comment]
1. I'm just applying this to the body field right now for all types, looking for 'a' and 'img'; 2. Most 'img' tags are going to a static location rather than images within the CMS; if we have a new static location for all of those files in Akumina, it would be easy to make the substitution with that now; 3. I noticed that there were a large number of links that went to '/node/NID' rather than the alias; I adjusted the XML exports to includes NIDs for all content, added a map of NIDs to new Akumina urls, and setup a lookup for that in the event hrefs are '/node/'.
